### PR TITLE
throw non zero exit status on failure of copy and move command

### LINKED ISSUE #251: Zbox move and copy returns a zero exit code on error
Zbox move and copy returns a zero exit code on error.
This could lead to a script or other orchestrator believing the operation was successful when it was not 

Fix: 
on ./zbox copy and ./zbox move, if an operation fails, return a non-zero exit code

### PR DESCRIPTION
**Pull request by boddumanohar**
A brief description of the changes in this PR:

fixes https://github.com/0chain/zboxcli/issues/251

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: 
- blobber:
- gosdk:
- system_test: https://github.com/0chain/system_test/pull/614
- zwalletcli: 
- Other: ...

### CODE REVIEW

File: cmd/copy.go [hunk 1]
```diff
@@ -32,15 +33,15 @@ var copyCmd = &cobra.Command{
 		allocationObj, err := sdk.GetAllocation(allocationID)
 		if err != nil {
 			fmt.Println("Error fetching the allocation", err)
```
Comment (cnlangzi) [inline cmd/copy.go]: maybe it is better to use panic?
```suggestion
panic(""Error fetching the allocation: "+err.Error())
```

**Review by cnlangzi — CHANGES_REQUESTED** [review]

**Comment by boddumanohar** [comment]
@cnlangzi  updated the PR as per comments. PTAL

**Comment by cnlangzi** [comment]
> @cnlangzi updated the PR as per comments. PTAL

`os.Exit(1)` can be removed , because panic will throw 1 now

**Review by cnlangzi — APPROVED** [review]
LGTM

**Review by cnlangzi — APPROVED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by cnlangzi** [comment]
@boddumanohar please also fix related system_tests, thanks.

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | file-copy  |
| gosdk  | staging  |
| zboxcli  | 251  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | file-copy  |
| gosdk  | staging  |
| zboxcli  | 251  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | staging  |
| zboxcli  | 251  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by peterlimg** [comment]
@boddumanohar please fix the system tests https://github.com/0chain/system_test/tree/file-copy.

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | file-copy  |
| gosdk  | staging  |
| zboxcli  | 251  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | file-copy  |
| gosdk  | staging  |
| zboxcli  | 251  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | file-copy  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**